source/tags/Icon/README.md

```diff
@@ -1,9 +1,15 @@
 # Icon
 
+Icons represent an instance of `<use xlink:href="#icon" />` which inserts a referance to the svg symbol in its place.
```

Comment:
methinks svg symbol should by wrapped in angle brackets and marked as `code`

(yeah, I said methinks. what of it?)